src/cli/commands/install.js

```diff
@@ -422,6 +423,11 @@ export class Install {
     }
 
     steps.push(async (curr: number, total: number) => {
+      this.reporter.step(curr, total, this.reporter.lang('fetchingPackages'), emoji.get('mag'));
```

Comment:
You should replace `fetchingPackages` with `checkingManifest` here - however I'm a bit concerned it will clutter the display. Could you add this step only if there's an `engine` field?